src/core/qfieldcloudprojectsmodel.h

```diff
@@ -429,6 +433,10 @@ class QFieldCloudProjectsModel : public QAbstractListModel
 
         QDateTime lastLocalDataLastUpdatedAt;
         QDateTime lastRefreshedAt;
+
+        bool autoPushEnabled = false;
+        int autoPushDelay = 30; // in minutes
```

Comment:
```suggestion
        int autoPushDelayMins = 30;
```
No need for the comment ;)

Btw, I would name it `autoPushIntervalMins`, it's not a delay?